library/src/test/java/com/datatorrent/lib/parser/XmlParserApplicationTest.java

```diff
@@ -68,7 +68,7 @@ public Boolean call() throws Exception
           return TupleCount == 1;
         }
       });
-      lc.run(10000);// runs for 10 seconds and quits
```

Comment:
It would be nice to rewrite the test to use the new embedded mode API. Also not clear to me why it would take 30s to see the expected result.

Reply:
Here is an example: https://github.com/apache/apex-malhar/blob/master/examples/enricher/src/test/java/org/apache/apex/examples/enricher/ApplicationTest.java

Reply:
Reverted the 30s change. It maybe better to do the launcher changes wholesale across all the tests separtely